src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -103,9 +104,11 @@ public SSTableWriter open(Descriptor descriptor,
     static class ReaderFactory extends SSTableReader.Factory
     {
         @Override
-        public SSTableReader open(Descriptor descriptor, Set<Component> components, TableMetadataRef metadata, Long maxDataAge, StatsMetadata sstableMetadata, SSTableReader.OpenReason openReason, SerializationHeader header)
+        public SSTableReader open(Descriptor descriptor, Set<Component> components, TableMetadataRef metadata,
+                                  Long maxDataAge, StatsMetadata sstableMetadata, SSTableReader.OpenReason openReason,
+                                  SerializationHeader header, IndexSummary summary, FileHandle dfile, FileHandle ifile, IFilter bf)
```

Comment:
nit: For now I guess we should at least put the arguments on separate lines.

Reply:
I extract `Builder` into a separate class and pass it as the only argument to `open`.. WDYT?

Reply:
I had actually almost suggested that earlier. WFM